portfolio/src/main/webapp/about.css

```diff
@@ -0,0 +1,132 @@
+html, body {
+    width: 100%;
+    height: 100%;
+    margin: 0;
+}
+
+#header {
+    display: flex;
+    flex-direction: row;
+    padding: 10px;
+    align-items: center;
+    background-color: rgb(192, 192, 192);
+    background-image: linear-gradient(rgb(192, 192, 192), rgb(129, 129, 129));
+}
+
+#header img {
+    height: 75px;
+    width: 100px;
+}
+
+#truth_and_lies_game {
```

Comment:
truth-and-lies-game
Make sure to use hyphens instead of underscores for CSS classes and IDs =)

Reply:
Changed to hyphens